BLSQ/bluesquare-components
# WC2 183: pass classname to table cell

Problem: we need to be able to style `Table` cells and rows directly

##Changes

- Add `cellProps`and `rowProps` props to `Table`. Pass them dow to `Row` and extract `className` to enable styling. Both props are a function, to allow for conditional styling



## How to test

On iaso: pull https://github.com/BLSQ/iaso/tree/WC2-172_deduplication_front_end. Check `DuplicateDetails.tsx`
